fdb-record-layer-core/src/main/java/com/apple/foundationdb/record/provider/common/RecordSerializer.java

```diff
@@ -115,14 +115,27 @@ M deserialize(@Nonnull RecordMetaData metaData, @Nonnull Tuple primaryKey,
         DECRYPT_SERIALIZED_RECORD("decrypt serialized record");
 
         private final String title;
+        private final String logKey;
         Events(String title) {
             this.title = title;
+            this.logKey = this.name();
+        }
```

Comment:
As a matter of practice, even for simple constructors, I try to have only a single "real" constructor that is invoked by all other constructors, so this would be:

    Events(String title) {
        this(title, null);
    }

    Events(String title, @Nullable String logKey) {
        this.title = title;
        this.logKey = logKey == null ? this.name() : logKey;
    }